test/runtests.jl

```diff
@@ -1,4 +1,61 @@
+# This file contains code that was formerly part of Julia. License is MIT: https://julialang.org/license
+
 using AbstractFFTs
 using Base.Test
 
-# TODO
+import AbstractFFTs: plan_fft
+import Base: A_mul_B!, *
+
+mutable struct TestPlan{T} <: AbstractFFTs.Plan{T}
+    region
+    pinv::TestPlan{T}
+    TestPlan{T}(region) where {T} = new{T}(region)
+end
+
+AbstractFFTs.plan_fft(x::Vector{T}, region; kwargs...) where {T} = TestPlan{T}(region)
+
+function Base.A_mul_B!(y::Vector, p::TestPlan, x::Vector)
+    n = length(x)
+    length(y) == n || throw(DimensionMismatch())
+    c = -2im * π / n
+    @inbounds for k = 0:n-1
+        y[k+1] = dot(x, exp.(c .* k .* (0:n-1)))
```

Comment:
Seems like it would be clearer (and faster) to just use a double loop here:
```jl
c = -2π/n
y .= 0
for k = 0:n-1, j=0:n-1
    y[k+1] += x[j+1] * cis(c*j*k)
end
```